packages/ember-input-validation/src/components/tpk-validation-radio.gts

```diff
@@ -28,27 +28,40 @@ export interface TpkValidationRadioComponentSignature
 }
 
 export default class TpkValidationRadioComponent extends BaseValidationComponent<TpkValidationRadioComponentSignature> {
-  @action onChange(value: string) {
+  @action onChange(value: string) {  
     if (this.args.onChange) {
       return this.args.onChange(value);
     }
-    return this.args.changeset.set(this.args.validationField, value);
+    const changeset = this.args.changeset.set(this.args.validationField, value)
+    return changeset;
   }
 
   get value() {
-    return super.value?.toString();
+    return this.args.value
+  }
+
+  get label() {
+    if(!this.args.label && !this.args.value) {
```

Comment:
une valeur comme label c'est rare, surtout que ce sera généralement traduit. Juste this.args.label me semble mieux

Reply:
ok j'etais plus generaliste, je ne pense pas qu'en terme de triptyk mais aussi des user externe mais j'ai retiré